packages/ses/README.md

```diff
@@ -74,8 +119,6 @@ c1.global === c2.global; // false
 c1.global.JSON === c2.global.JSON; // true
```

Comment:
`c1.global` or `c1.globalThis`? I guess it depends upon whether #313 lands before or after this one.

Reply:
This is the PR that resolves #313, so I’ll fix this here!